TestPlaybooks/playbook-RecordedFutureFeed.yml

```diff
@@ -0,0 +1,669 @@
+id: RecordedFutureFeed - Test
```

Comment:
Create a new directory in your pack called `TestPlaybooks`. Then, please move this file to that directory, and remove the `playbook-` prefix from the filename.